app/scripts/modules/core/src/utils/stickyHeader/stickyHeader.less

```diff
@@ -17,6 +23,18 @@
   .clearfix();
 }
 
+.sticky-header-2 {
```

Comment:
* `sticky-header-2` is always used for a header in a `pod-subgroup`, so maybe `.sticky-header-pod-subgroup`? Or since it's always under a `pod-subgroup` class and it's not a conditionally applied class, just create a class `pod-subgroup-header` and make that class set the sticky values?
* `sticky-header-3` is only used in one place, so that could just be directly named like `.sticky-header-server-group-title`